lib/ansible/modules/files/file.py

```diff
@@ -173,8 +173,87 @@
     modification_time: now
     access_time: now
 '''
-RETURN = '''
 
+RETURN = r'''
+changed:
```

Comment:
changed/failed/skipped are 'common return fields' and don't need to be documented  in each plugin

Reply:
@bcarroll common return values are documented [here](https://docs.ansible.com/ansible/latest/reference_appendices/common_return_values.html#common-return-values)
When you build your module docs, the `RETURN` section should include a pointer to that page of common return values documentation just under the header. You can test this locally - follow the instructions at https://docs.ansible.com/ansible/devel/dev_guide/testing_documentation.html.